tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -664,7 +656,7 @@ module rec CompilerAssertHelpers =
             { new IDisposable with
               member _.Dispose() = try File.Delete runtimeconfigPath with | _ -> () }
 #endif
-        let timeout = 30000
+        let timeout = 60000
```

Comment:
From my experience, increasing timeouts in tests is rarely effective in the long run. Timeouts usually mean badly designed code, or badly designed tests, or both - so it might be rather just rethink what's going on (not necessarily in this PR)